frontend/src/Student/components/PortfolioOtherMediaForm.js

```diff
@@ -0,0 +1,344 @@
+import React, { Component, Fragment } from "react";
```

Comment:
I believe so. https://trello.com/c/3rykDISQ/83-decouple-entry-from-peice is about making Entry and Piece decoupled, so it would enable the component refactors.